File: 9. Palindrome Number/Solution.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
        if x == 0:
            return True
        l = []
        while x != 0:
            pop = x % 10
            x //= 10
            l.append(pop)
        i = 0
        j = len(l) - 1
        while i <= j:
            if l[i] != l[j]:
                return False
            i += 1
            j -= 1
        return True

File: 9. Palindrome Number/Solution.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0)
            return false;
        if (x == 0)
            return true;
        vector<int> n;
        while (x != 0) {
            int pop = x % 10;
            x /= 10;
            n.push_back(pop);
        }
        for (int i = 0, j = n.size() - 1; i <= j; i++, j--) {
            if (n[i] != n[j])
                return false;
        }
        return true;
    }
};